ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -1906,6 +1906,38 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: ipi-openstack
+- as: azure-ipi-fips-p3-f2-openshift-logging-5-5
+  cron: 00 21 * * 1,3,5
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      CO_SUB_CHANNEL: stable-5.5
+      COMPUTE_NODE_TYPE: Standard_D4s_v3
+      E2E_RUN_TAGS: '@logging5.6'
```

Comment:
@IshwarKanse  Why we use logging5.6 tag here?

Reply:
@anpingli It was just to test the rehearsal job before we get the '@logging5.5' tag prs in cucushift and verification-tests repo merged (The rehearsal jobs can take hours to complete). I've updated this pr to use the '@logging5.5'  tag.